### PR TITLE
Release version 1.3

### PR DESCRIPTION
**Pull request by HajimeKawahara**
This is the release branch for v1.3.
We plan to release v1.3 March 16th before ASJ meeting. 

CI fails due to (maybe) vaex related error. So, try unit tests. 
Now, I wrote some documents on new features and reorganize the structure of the documents: 
http://secondearths.sakura.ne.jp/exojax/develop/
Check it. Maybe it's not enough yet (for instance, the documents for OpaDirect, OpaMODIT have not yet been written. 
I will try to write them. 

- v1.3
https://github.com/HajimeKawahara/exojax/milestone/5

### CODE REVIEW

**Comment by chonma0ctopus** [comment]
Thank you for the PR. I confirmed that all the unit tests work and the reverse_premodit.py at least started to run HMC.

**Review by YuiKasagi — APPROVED** [review]
Thank you for the update! All unit tests worked in my environment. I modified the optimization code with jaxopt I made before to use this version, and it works with no problem.

**Review by astrostevanus — APPROVED** [review]
Everything works well!

File: tests/endtoend/reverse/reverse_modit.py [hunk 1]
```diff
@@ -4,149 +4,149 @@
...
 rng_key = random.PRNGKey(0)
 rng_key, rng_key_ = random.split(rng_key)
 num_warmup, num_samples = 300, 600
-#kernel = NUTS(model_c, forward_mode_differentiation=True)
+# kernel = NUTS(model_c, forward_mode_differentiation=True)
 kernel = NUTS(model_c, forward_mode_differentiation=False)
 
 mcmc = MCMC(kernel, num_warmup=num_warmup, num_samples=num_samples)
-mcmc.run(rng_key_, nu1=nusd, y1=nflux)
+mcmc.run(rng_key_, y1=nflux)
```
Comment (chonma0ctopus) [inline tests/endtoend/reverse/reverse_modit.py:149]: Also in line 154, nu1 should be removed as `predictions = pred(rng_key_, y1=None)` although I'm afraid a very minor correction.

Comment (HajimeKawahara) [reply tests/endtoend/reverse/reverse_modit.py:149]: thanks!

Comment (HajimeKawahara) [reply tests/endtoend/reverse/reverse_modit.py:149]: corrected.

**Review by ykawashima — APPROVED** [review]
Thank you very much! I apologize for the delay in reviewing. I have confirmed that all the unit tests work in my environment, too, except for [this](https://github.com/HajimeKawahara/exojax/blob/a04e627ba51b669501999605ee88eff1a596ae33/tests/unittests/spec/xs/f64/ljert_f64_test.py). But I think we can ignore this since I just got the difference of 6.4e-5 while the criterion is 6e-5.

**Pull request merged**